lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -871,6 +1266,86 @@ void HdVP2Mesh::_InitRepr(const TfToken& reprToken, HdDirtyBits* dirtyBits)
     }
 }
 
+void HdVP2Mesh::_CreateSmoothHullRenderItems(HdVP2DrawItem& drawItem)
+{
+    // 2021-01-29: Changing topology is not tested
```

Comment:
Do we have workflows where topology changes?

Reply:
Currently with the consolidation update workaround, if we modify a shader attribute value, we will send DirtyMaterialId message to the Rprim side which will then propagate the dirty bits to topology.